docs/Vectors.md

```diff
@@ -139,6 +139,42 @@ FT.CREATE ... SCHEMA ... {field_name} VECTOR {algorithm} {count} [{attribute_nam
     EF_RUNTIME 20
     ```
 
-<!-- ## Querying vector fields
+## Querying vector fields
 
-TO BE ADDED -->
+We allow using vector similarity queries in the `FT.SEARCH` "query" parameter. The syntax for vector similarity queries is `*=>[{vector similarity query}]` for running the query on an entire vector field, or `{some sub-query}=>[{vector similarity query}]` for running similarity query on the result of the sub-query.
+
+As of version 2.4, we allow vector similarity to be used **once** in the query, and on the entire other query parts as its filter.
```

Comment:
This is a bit unclear. I suggest we leave it as
As of version 2.4, we allow vector similarity to be used once in the query.

Reply:
I tried to explain that we don't allow `"*=>[{vector query}] AND @n:[1 100]"` for example